src/app/track/track.module.ts

```diff
@@ -1,6 +1,7 @@
 import { ModuleWithProviders, NgModule } from '@angular/core';
 import { RouterModule } from '@angular/router';
 import { LuxonModule } from 'luxon-angular';
+import 'ol-layerswitcher/dist/ol-layerswitcher.css';
```

Comment:
Is importing a css file in the TypeScript module really necessary? It's imported in the general css file anyway?